### PR TITLE
(work-around) Exclude text file 'text!js-translation.json' from bundling

### LINKED ISSUE #47: Magento 2.3.4 JS-Translations fall back to default store language
In a multistore setup with different languages, some translations like for "Total" in checkout cart or the ones for the "add to cart" button, just after clicking it, fall back to the ones given for the default store view.

This only happens with the Baler module being enabled.

A colleague of mine debugged it to magento/module-translation/view/base/web/js/image-translation-dictionary.js

This was introduced in the following commit:
https://github.com/magento/magento2/commit/71c781df5536a2ec79c9cf42ade0c262396767a3
Which was seen as a resolution for
https://github.com/magento/baler/issues/13

### PR DESCRIPTION
**Pull request by tdgroot**
This fixes #47.

Now, I know that this is not the most thorough solution, but this gets my build going again. Feel free to give feedback/merge/decline!

### CODE REVIEW

**Comment by DrewML** [comment]
Thanks @tdgroot! This looks great.

Would you be willing to add a test to cover this? We can add one in [`computerDepsForBundle.unit.js`](https://github.com/magento/baler/blob/a89a97feca9c3287c3bcc1c25d591f5177f3e665/src/__tests__/computeDepsForBundle.unit.js), and the associated fixture in [`src/__tests__/__fixtures__`](https://github.com/magento/baler/tree/master/src/__tests__/__fixtures__).

File: src/ignoredTextFiles.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
```
Comment (DrewML) [inline src/ignoredTextFiles.ts]: Thoughts on changing this file to something like `knownIgnores` or `excludedFiles`? We don't discriminate text vs other resource type in `computeDepsForBundle`, so we can probably just make use of this file for any future Magento-specific ignores we have.

Comment (tdgroot) [reply src/ignoredTextFiles.ts]: Good point, I renamed the file!

**Comment by tdgroot** [comment]
> Thanks @tdgroot! This looks great.
> 
> Would you be willing to add a test to cover this? 
>

Done!

**Comment by tdgroot** [comment]
Thank you for your review! :+1: I added the requested changes.

**Comment by DrewML** [comment]
Thanks @tdgroot! Appreciate the contribution 😃

**Pull request merged**